services/brig/src/Brig/Options.hs

```diff
@@ -519,6 +524,26 @@ data CustomerExtensions = CustomerExtensions
 newtype DomainsBlockedForRegistration = DomainsBlockedForRegistration [Domain]
   deriving newtype (Show, FromJSON, Generic)
 
+data SFTOptions = SFTOptions
+  { sftBaseDomain :: !DNS.Domain,
+    sftSRVServiceName :: !(Maybe ByteString), -- defaults to defSftServiceName if unset
+    sftDiscoveryIntervalSeconds :: !(Maybe DiffTime) -- defaults to defSftDiscoveryIntervalSeconds
+  }
+  deriving (Show, Generic)
+
+instance FromJSON SFTOptions where
```

Comment:
(Wild thought: we should have a couple of strings in the test suite and tests that they parse as server config files.)